src/utils/HTMLUtils/index.js

```diff
@@ -6,14 +6,26 @@ const isPlainText = elem => typeof elem === "string" && !elem.startsWith("<");
 
 const getInnerHTML = elem => (isPlainText(elem) ? elem : cheerio(elem).html());
 
+const removeDash = elem => replace(/-/g, "_", elem);
+
 const unescapePiping = value =>
   replace(
     /{{([^}}]+)}}/g,
     (_, match) => `{{${replace(/&apos;/g, "'", match)}}}`,
     value
   );
 
-const getInnerHTMLWithPiping = elem => unescapePiping(getInnerHTML(elem));
+const getInnerHTMLWithPiping = elem => {
+  if (!elem) {
+    return;
+  }
+
```

Comment:
Should this throw an error instead of return undefined? Just thinking about when it comes to debugging later on.

Reply:
I thought it should at first and at one point I did include them. If you throw an error it stops the whole process and you can't see where the error is coming from. As you don't know if it's coming from `section.title` or `question.title` because the error just refers to what line of code is broken.
Returning undefined means that the validator can notice what's missing and tell you specifically that section requires a title and there isn't one present.

Reply:
Throwing an error instead of returning undefined might be a huge task. The Publisher tests run on the basis that not all parts of the questionnaire are present. Throwing an error basically outputs a million logs to the test console.

I've added an extra commit to highlight this. @MattElcock could you check it over and let me know what you think? Thanks

Reply:
Hmm, I see. I s'pose, too, if `elem` is undefined there'll be no way to identify what questionnaire the log belongs to in production, so those million logs won't really be overly useful other than locally...

I'm happy to leave it as returning `undefined` then. Cheers for looking into this!